test/services/apimap-field-builder.test.js

```diff
@@ -12,47 +12,181 @@ const sequelize = new Sequelize(
   databaseOptions,
 );
 
-async function initializeField() {
-  const model = sequelize.define('user', {
-    uuid: {
-      type: Sequelize.DataTypes.UUID,
-      defaultValue: Sequelize.DataTypes.UUIDV4,
-    },
-  });
+async function initializeField(fieldDefinitions, modelName = 'testModel') {
+  const model = sequelize.define(modelName, fieldDefinitions);
 
   return sequelize.sync({ force: true })
-    .then(() => new ApimapFieldBuilder(
-      model,
-      _.values(model.rawAttributes)[1],
-      { sequelize: Sequelize },
-    )
-      .perform());
+    .then(() => _.mapValues(model.rawAttributes, (attribute) =>
+      new ApimapFieldBuilder(
+        model,
+        attribute,
+        { sequelize: Sequelize },
+      )
+        .perform()));
 }
 
 describe('services > apimap-field-builder', () => {
   describe('on a UUID column with a UUIDV4 defaultValue', () => {
+    const fieldDefinitions = {
+      uuid: {
+        type: Sequelize.DataTypes.UUID,
+        defaultValue: Sequelize.DataTypes.UUIDV4,
+      },
+    };
+
     it('should have a name uuid', async () => {
       expect.assertions(1);
-      const field = await initializeField();
-      expect(field.field).toStrictEqual('uuid');
+      const { uuid } = await initializeField(fieldDefinitions);
+      expect(uuid.field).toStrictEqual('uuid');
     });
 
     it('should have a String type', async () => {
       expect.assertions(1);
-      const field = await initializeField();
-      expect(field.type).toStrictEqual('String');
+      const { uuid } = await initializeField(fieldDefinitions);
+      expect(uuid.type).toStrictEqual('String');
     });
 
     it('should not be set as required', async () => {
       expect.assertions(1);
-      const field = await initializeField();
-      expect(field.isRequired).toStrictEqual(false);
+      const { uuid } = await initializeField(fieldDefinitions);
+      expect(uuid.isRequired).toStrictEqual(false);
     });
 
     it('should not have a default value', async () => {
       expect.assertions(1);
-      const field = await initializeField();
-      expect(field.defaultValue).toBeUndefined();
+      const { uuid } = await initializeField(fieldDefinitions);
+      expect(uuid.defaultValue).toBeUndefined();
+    });
+  });
+
+  describe('on other default values', () => {
+    it('should handle raw values', async () => {
```

Comment:
[Suggestion] Maybe we could have done two tests to make it clearer, in other words:

```js
describe('on raw default values', () => {
  it('should handle integer values', async () => {});
  it('should handle string values', async () => {});
});
```

Or to be consistant with other tests:

```js
describe('on a integer column with a integer defaultValue', () => {
  it('should have an integer type', async () => {});
  // ...
});

describe('on a string column with a string defaultValue', () => {
  it('should have an string type', async () => {});
  // ...
});

```

Reply:
Done